tests/conftest.py

```
@@ -29,7 +34,31 @@
+from src.main_app import create_app, ensure_all_tables  # noqa: E402
 from src.main_app.api_services.mwclient_page import MwClientPage  # noqa: E402
+from src.main_app.config import DbConfig, TestingConfig  # noqa: E402
+from src.main_app.db.engine_sqlite import DatabaseSqlLite  # noqa: E402
+
+
+@pytest.fixture(autouse=True)
+def mock_initialize_db(monkeypatch: pytest.MonkeyPatch):
+    ensure_all_tables(None, DatabaseSqlLite())
+
+    def _mock(_db_class):
+        database_data = DbConfig(db_host="localhost", db_name="test", db_user="user", db_password="pass")
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "db_password"

(S106)